frontend/src/mocks/utils.ts

```diff
@@ -16,8 +16,11 @@ export const mocksToHandlers = (mocks: Mocks): ReturnType<typeof rest['get']>[]
             response.push(
                 (rest[method] as typeof rest['get'])(path, async (req, res, ctx) => {
                     if (typeof handler === 'function') {
-                        const [status, resp] = handler(req, res, ctx)
-                        return res(ctx.status(status), ctx.json(resp ?? null))
+                        const responseArray = handler(req, res, ctx)
```

Comment:
Something I liked about the other solution was:

 1. we had both request and response type safety, tied specifically to the same types used for the api client. If we change something in the api client for the boundary types, we will be sure that any fixtures will be updated by the dev also, thereby avoiding stale stories/tests.
 2. we reuse the express handler interface and avoid creating our own, which will need maintenance, documenting etc. It seems like an unnecessary abstraction.
 3. we decouple the url path from the handler, which I don't think is ever relevant

Just my 2c